.github/workflows/run-codeql.yaml

```diff
@@ -1,26 +1,32 @@
-name: "Code Scanning"
+name: CodeQL Scan
 
 on:
   push:
-    branches: [master]
+    branches: 
+      - master
   pull_request:
-    branches: [master]
+    branches:
+      - master
   schedule:
     - cron: '30 1 * * 0'
 
 jobs:
-  CodeQL-Build:
+  codeql-scan:
+    name: CodeQL Scan
```

Comment:
If we got this way, we might better use just `Scan` because with `CodeQL Scan` our build status title ends up with `CodeQL Scan / CodeQL Scan` which isn't clear which is the workflow name or the job name, whereas with `Scan` it might be like `CodeQL / Scan`.